tree/dataframe/test/dataframe_simple.cxx

```diff
@@ -513,6 +514,113 @@ TEST_P(RDFSimpleTests, BookCustomAction)
    EXPECT_EQ(*maxSlot, nWorkers-1);
 }
 
+class StdDevHelper {
+private:
+   std::default_random_engine fGenerator;
+   std::normal_distribution<double> fDistribution;
+
+public:
+   std::vector<double> GenerateNumbers(int n)
+   {
+      std::vector<double> numbers;
+      for (int i = 0; i < n; ++i)
+         numbers.push_back(fDistribution(fGenerator));
+      return numbers;
+   }
+
+   double stdDevFromDefinition(const std::vector<double> &samples)
+   {
+      // Calculating the Variance using the definition
+      int nSamples = samples.size();
+      double mean = 0;
+      for (int i = 0; i < nSamples; ++i) {
+         mean += samples[i];
+      }
+      mean = mean / nSamples;
+
+      double varianceRight = 0;
+
+      for (int i = 0; i < nSamples; ++i) {
+         varianceRight += std::pow((samples[i] - mean), 2);
+      }
+      varianceRight = varianceRight / (nSamples - 1);
+      return std::sqrt(varianceRight);
+   }
+
+   double stdDevFromWelford(const std::vector<double> &samples)
+   {
+      ROOT::RDataFrame d(samples.size());
+      return *d.DefineSlotEntry("x",
+                                [&samples](unsigned int slot, ULong64_t entry) {
+                                   (void)slot;
+                                   return samples[entry];
+                                })
+                 .StdDev("x");
+   }
+};
+
+TEST(RDFSimpleTests, StandardDeviation)
```

Comment:
you should use `TEST_P(RDFSimpleTests` instead of `TEST`, so the test will use the `RDFSimpleTests` fixture defined at the beginning of the file and will run in single and multi-thread.